qa/L0_model_config/test.sh

```diff
@@ -182,6 +182,19 @@ for modelpath in \
        $modelpath/.
 done
 
+# Copy Python models into the test model repositories.
+for modelpath in \
+        autofill_platform/python/mismatch_datatype/1 \
+        autofill_platform/python/mismatch_dims/1 \
+        autofill_platform/python/unknown_input/1 \
+        autofill_platform/python/unknown_output/1 \
+        autofill_platform_success/python/empty_config/1 \
+        autofill_platform_success/python/incomplete_input/1 \
```

Comment:
I meant these two here. You can do something like:

incomplete_input : <incomplete_config> + <full input/ouptut config in auto_complete_config> == This will be like you have now.

incomplete_output:  <incomplete_config> + <just the missing config in auto_complete_config>

Reply:
Please correct me if I'm wrong, do you mean that, for example, if INPUT0 is missing dims property, then in model.py we just provide dims property for INPUT0? But our API has a checking to make sure that users add all name, datatype and dims properties when setting the configuration, we cannot only provide a specific property for a input/output.

Reply:
If we provide only INPUT0 in config.pbtxt and INPUT1 in model.py, I don't think this will pass the logic for checking allowed input/output name here: https://github.com/triton-inference-server/python_backend/blob/krish-python/src/python.cc#L2506
Do we want to remove this checking?

Reply:
@krishung5  I see. Let's keep the check for now for consistency with other backends.  We can relax it in future if required.

@krishung5 @Tabrizian  Do you think it will be a value add for users to not specify every input in `auto_complete_config` model.py?
Unlike other models. the model.py execution will not be affected by extra inputs/outputs in request/response.

Reply:
Do you mean allowing users to add some extra inputs/outputs that are not used in the model.py execution? So if they use `curl localhost:8000/v2/models/<model name>/config`, they will see the extra inputs/outputs in the configuration? I think this should be fine since it won't affect any request/response like you mentioned, as long as this won't confuse users at any point?

Reply:
I think after the discussion in [here](https://github.com/triton-inference-server/python_backend/pull/138#discussion_r868251707) it is allowed to have unknown inputs since it will be auto-completed by the backend.